app/src/main/res/values/strings.xml

```diff
@@ -84,4 +84,10 @@
     <string name="digital_outputs">Digital Outputs</string>
     <string name="configure_PWM">Configure PWM</string>
     <string name="waveform_generator">Arbitrary Waveform Generator</string>
+    <string name="autoscan">AUTOSCAN</string>
+    <string name="select_sensor">SELECT SENSOR</string>
+    <string name="get_raw">GET RAW</string>
+    <string-array name="sensors">
+
```

Comment:
There is a layout rendering error if I don't reference a string array in the spinner.